gulpfile.babel.js

```diff
@@ -161,8 +161,7 @@ gulp.task('html', () => {
 });
 
 // Clean output directory
-gulp.task('clean', cb => del(['.tmp', 'dist/*', '!dist/.git'], {dot: true},
-  cb));
+gulp.task('clean', () => del(['.tmp', 'dist/*', '!dist/.git'], {dot: true}));
```

Comment:
Just to help the future PR that fix that, this is the PR who added `, {dot: true}:` #537